plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/LibvirtComputingResource.java

```diff
@@ -3300,7 +3301,7 @@ public boolean configureNetworkRulesVMSecondaryIP(final Connect conn, final Stri
             return false;
         }
 
-        final Script cmd = new Script(_securityGroupPath, _timeout, s_logger);
+        final Script cmd = new Script(_securityGroupPath, _timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.